docs/blog/Webots-2023-a-release.md

```diff
@@ -0,0 +1,98 @@
+
+%figure "New devices"
+![Added Devices](images/devices_r2023a.thumbnail.png)
+%end
+
+---
+
+## New Assets
+
+Additional models have been added to the Webots library, namely static animals (cow, horse, deer, sheep, dog, fox, cat and rabbit) and a barn.
+
+%figure "New objects"
+![Added Assets](images/assets_r2023a.thumbnail.png)
+%end
+
+---
+
+## New Python API
+
+Webots R2023a comes with a brand new Python API! This API has the advantage of being compatible with all Python distributions (including conda, anaconda, etc.). 
+The API will always be compatible with current and upcoming Python versions.
+
+This change of API does not induce any change to the methods. 
+Python controllers developed before Webots R2023a are fully compatible with this new version.
+
+Moreover, in order to enrich the collection of sample controllers, all sample worlds of the device nodes provide an additional Python controller equivalent to the C controller already included to demonstrate the API functions.
+
+---
+
+## New IMU Sample World
+
+So far, the [InertialUnit](../reference/inertialunit.md) was considered as an IMU, because it provides the ground truth data of the attitude (roll, pitch, yaw) of the object on which it is grafted. 
+This assumption is fundamentally wrong.
+An IMU gives the raw values of its sensors ([Accelerometer](../reference/accelerometer.md), [Gyroscope](../reference/gyro.md), [Compass](../reference/compass.md)) and can optionally provide the attitude output computed from the fusion of its sensors. 
+The [InertialUnit](../reference/inertialunit.md) just makes life easier for the user who does not want to use a sensor fusion library or algorithm to combine the data from an IMU and compute the attitude.
+
+In addition to the implementation of the [Mpu-9250](../guide/imu-sensors.md#mpu-9250) IMU, Webots R2023a contains a new sample world that demonstrates a simple algorithm for calculating the attitude of a robot from the sensors of an IMU.
+The results of the latter are compared to the values given by the [InertialUnit](../reference/inertialunit.md).
+
+You can explore this new sample world in its demo world under `File > Open Sample World > imu.wbt`.
+
+---
+
+## ROS2
+
+With this release we are continuing to develop and improve the support for ROS 2 in Webots.
+New features include:
+
+- Added support for Windows Subsystem for Linux (WSL) and macOS.
+- Added reset handler to all examples of [`webots_ros2`](https://github.com/cyberbotics/webots_ros2) to support simulation reset from Webots.
+
+And many bug fixes.
+
+---
+
+## Extra Goodies
+
+- A new non-interactive terminal for web streaming.
+- The possibility to choose between spherical and cylindrical projections in [Camera](../reference/camera.md), [RangeFinder](../reference/rangefinder.md), and [Lidar](../reference/lidar.md) nodes.
```

Comment:
The Lidar doesn't have the possibility to have a spherical projection:
```suggestion
- The possibility to choose between spherical and cylindrical projections in [Camera](../reference/camera.md) and [RangeFinder](../reference/rangefinder.md) nodes.
```

Reply:
Oh yes, I misread the changelog.